gravitee-apim-console-webui/src/management/settings/client-registration-providers/client-registration-provider/client-registration-provider.component.html

```diff
@@ -70,7 +70,17 @@ <h2>Configuration</h2>
 
         <mat-form-field appearance="outline" class="client-registration-provider__card__content__form-field">
           <mat-label>Client Secret</mat-label>
-          <input type="text" matInput formControlName="client_secret" required />
+          <input matInput [type]="hideClientSecretPassword() ? 'password' : 'text'" formControlName="client_secret" required />
+          <button
+            mat-icon-button
+            matSuffix
+            type="button"
+            [attr.aria-label]="'Hide password'"
+            (click)="clickHideClientSecretPassword($event)"
+          >
+            <mat-icon class="gio-metadata-dialog__title__cancel" [svgIcon]="hideClientSecretPassword() ? 'gio:eye-empty' : 'gio:eye-off'">
+            </mat-icon>
+          </button>
```

Comment:
I seem to remember that this should be enough, no need to add anything to component.ts, i'm right ? 🤔 

```suggestion
          <input matInput #clientSecretInput formControlName="client_secret" required />
          <button
            mat-icon-button
            matSuffix
            type="button"
            [attr.aria-label]="'Hide password'"
            (click)="clientSecretInput.type = clientSecretInput.type === 'password' ? 'text' : 'password'"
          >
            <mat-icon class="gio-metadata-dialog__title__cancel" [svgIcon]="clientSecretInput.type === 'password' ? 'gio:eye-empty' : 'gio:eye-off'">
            </mat-icon>
          </button>
```